target-platform/p2-repo-common/pom.xml

```diff
@@ -82,6 +82,11 @@
 									<version>${org.eclipse.kura.sun.misc.version}</version>
 								</artifactItem>
 								<artifactItem>
+								    <groupId>org.json</groupId>
```

Comment:
So, as agreed on EclipseCon, go ahead with org.json as long as we replace this dependency before 2.2 will be released.